src/components/mainContent/PostsList.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Post from "../posts/Post";
+import CommentSection from "../comments/CommentSection";
+
+export default function PostsList(props) {
+  let ourContent = [];
+
+  /*This function takes the list of post/followups from our backend and renders it accordingly
+Some post sections might not have followups or vice versa, which is why it checks for null*/
+  props.content.forEach((section) => {
```

Comment:
In this case, it won't explicitly be one-to-one since each post might have a followup as well

Reply:
We can always restructure it so that it can be a one-to-one thing and refactor, which will probably be the case if we want to insert interactive visuals between the post and comment section

Reply:
Actually, we can just insert the visuals into the post by putting it inside of the body text and having it get rendered.